src/nopoll_decl.h

```diff
@@ -216,7 +216,8 @@
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
-#include <sys/select.h>
+/* #include <sys/select.h> */
+#define NOPOLL_HAVE_POLL 1
```

Comment:
this should be automatically detected with autoconf if not then we should add the test to configure script.

Reply:
I will try without the #define.